common/source/docs/common-autopilots.rst

```diff
@@ -64,6 +66,8 @@ Closed hardware
     VR Brain 5 <http://www.virtualrobotix.it/index.php/en/shop/autopilot/vrbrain5-detail>
     VR uBrain 5.1 <http://www.virtualrobotix.it/index.php/en/shop/autopilot/vrbrainmicro51-detail>
 
+\^ due to flash memory limitations, these boards do not include all ArduPilot features. See :ref:`Firmware Limitations <common-autopilots_limited_firmware>` for details.
```

Comment:
I would prefer using the ** notation for the second note instead of ^....looks a bit awkward to me...personal perference

Reply:
Fixed to **